front/src/pages/accounts.jsx

```diff
@@ -0,0 +1,15 @@
+import { useContext } from 'react';
+import { AccountsContext } from '../context/accounts-context';
+import AccountList from '../components/Accounts/Accounts/AccountList';
+
+export default function Accounts() {
+	const { accounts } = useContext(AccountsContext);
+
+	if (accounts.length === 0) return <h1>Loading...</h1>;
```

Comment:
for new users, accounts will be empty list, but user will see `Loading...` even after its loaded.
Please show some other message when there is no accounts. Show loading message for only while loading.